railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
I think this belongs more to bundler global config than to Rails itself. It
does not make sense to us use all the cores on rails new and not in all
subsequent bundle install. I'm personally -1 about introducing this code to
Rails.

On Sun, Apr 26, 2015, 11:02 Carlos Antonio da Silva <
notifications@github.com> wrote:

> In railties/lib/rails/generators/app_base.rb
> <https://github.com/rails/rails/pull/19908#discussion_r29110058>:
>
> > @@ -340,7 +340,15 @@ def spring_install?
> >        end
> >
> >        def run_bundle
> > -        bundle_command('install') if bundle_install?
> > +        # Most of us have the CPUs; so let's parallelize `bundle install`
>
> [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/19908/files#r29110058>.
>